schema/cosa/schema_doc.go

```diff
@@ -1,4 +1,4 @@
-// Generated by ./generate-schema.sh
+// Generated by /home/rmohr/coreos-assembler/schema/generate-schema.sh
```

Comment:
Probably not wanted.

Reply:
Fixed. :+1: 

I had some left overs on some schema change experiments in the PR. Removed them all now.